lib/src/collect.dart

```diff
@@ -110,19 +112,22 @@ Future<Map<String, dynamic>> _getAllCoverage(
         if (!scopedOutput.contains(scope)) continue;
         final scriptReport = await service.getSourceReport(
             isolateRef.id!, <String>[SourceReportKind.kCoverage],
-            forceCompile: true, scriptId: script.id);
-        final coverage = await _getCoverageJson(
-            service, isolateRef, scriptReport, includeDart, functionCoverage);
+            forceCompile: true,
+            scriptId: script.id,
+            reportLines: reportLines ? true : null);
```

Comment:
Why does this argument take `true` or `null` and not `true` or `false`?

Reply:
If the service implementer doesn't support the reportLines flag, I don't want to send it at all. The VM ignores unrecognized flags, but I'm not sure about the other backends.